Query/Delete.php

```diff
@@ -75,6 +75,7 @@ public function from($source)
      */
     public function __toString()
     {
-        return 'DELETE FROM ' . $this->_from . parent::__toString();
+        return 'DELETE FROM ' . $this->enclose($this->_from) .
```

Comment:
CS:
```php
return
    'DELETE FROM ' .
    $this->_from .
    parent::__toString();
```
When we have multiple concatenation, we prefer this syntax now :-). Had to be formalized though…